src/services/PostgresDatabaseService.ts

```diff
@@ -186,11 +184,11 @@ export default class PostgresDatabaseService implements IDatabaseService {
         return values.map(([prop], index) => prop + ' = $' + (index + offset)).join(', ');
     }
 
-    #deconstructValues(values: [string, any][]): Value[] {
+    #deconstructValues(values: [string, any][]): any[] {
```

Comment:
Ewww, any

Reply:
I am totaly with you on that regard! Apparently, ts-postgres was updated and they removed the `Value` type!
I had no intention to edit the bit in the beginning but I was forced to 😞